### ISSUE TITLE
Improve tuner performance with weight sharing

### ISSUE THREAD

**Issue by haifeng-jin** [keras-team/autokeras#916]

**Comment by haifeng-jin** [comment]
The weight sharing may negatively impact the performance like using pretrained models but tuning learning rate.
